scripts/verify

```diff
@@ -55,6 +55,8 @@ done
 [[ -z "$parameters" ]] && parameters="{}"
 [[ -z "$wait_timeout" ]] && wait_timeout=600
 
+error_summary_path="error_summary.txt"
```

Comment:
can we fix the location instead of having this in a relative path, like `/tmp/error_summary`? or create a temporary file and store the value here using `mktemp`